node/common/transport/amqp/lib/amqp.js

```diff
@@ -79,7 +80,7 @@ Amqp.prototype.connect = function connect(done) {
   }
 
   this._connectPromise
-    .then(function () {
+    .then(function (res) {
```

Comment:
it will, it's just remnants of when I was fleshing out what to do to have a transport-agnostic  "ConnectionSuccessful" object that would contain the transport-specific object itself (res)